src/network/invthread.py

```diff
@@ -106,6 +106,6 @@ def run(self):  # pylint: disable=too-many-branches
                 invQueue.task_done()
 
             if dandelion.instance.refresh < time():
-                dandelion.instance.reRandomiseStems()
+                dandelion.instance.reRandomiseStems(connectionpool.pool)
```

Comment:
this check could also be moved down (i.e. `reRandomiseStems` could check check the time.